libs/game.js

```diff
@@ -75,7 +76,7 @@ function check(levelObj) {
 }
 
 function correct(levelObj) {
-  console.log("\nWooo! You completed this git challenge! Run `Gitfun` to proceed to the next level.\n");
+  console.log(chalk.green("\nWooo! You completed this git challenge! ") + "Run `Gitfun` to proceed to the next level.\n");
```

Comment:
I just noticed another typo here — not urgent but you might want to hotfix it. `Gitfun` -> `gitfun`